cypress/support/api_commands.js

```diff
@@ -373,10 +373,17 @@ Cypress.Commands.add('apiUpdateConfig', (newSettings = {}) => {
     cy.apiLogout();
 });
 
+Cypress.Commands.add('apiGetConfig', () => {
+    cy.apiLogin('sysadmin');
+
+    // # Get current settings
+    return cy.request('/api/v4/config');
+});
+
 // *****************************************************************************
 // Post creation
 // *****************************************************************************
 
 Cypress.Commands.add('postMessageAs', (sender, message, channelId, createAt = 0) => {
-    cy.task('postMessageAs', {sender, message, channelId, createAt}).its('message').should('equal', message);
+    cy.task('postMessageAs', {sender, message, channelId, createAt});
```

Comment:
I removed the test as I don't think it's necessary.  Let the spec using this command do the test.

Reply:
We may want something chained off of the task command so that we get the nice auto waiting in cypress. `cy.task` may just fire off and and then continue on with the rest of the test without waiting for an actual response, so depending on response times tests it may impact tests. Maybe we can just check for status of 200?

Reply:
Yup but I'll go ahead for status check and then update in the future as needed.